lib/resources/example_json/r4_examples_practitioner.rb

```diff
@@ -0,0 +1,172 @@
+module Cerner
+  module Resources
+
+    R4_PRACTITIONER_ENTRY ||= {
+      "resourceType": "Practitioner",
+      "id": "1994021",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2013-11-04T13:25:17.000-06:00"
+      },
+      "text": {
+        "status": "generated",
+        "div": "&lt;div&gt;&lt;p&gt;&lt;b&gt;Practitioner&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Name&lt;/b&gt;: PCTAUT, OrdersPhy3&lt;/p&gt;&lt;p&gt;&lt;b&gt;Identifiers&lt;/b&gt;: DOCDEA: 234234123&lt;/p&gt;&lt;p&gt;&lt;b&gt;Gender&lt;/b&gt;: Male&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;/div&gt;"
+      },
+      "identifier": [
+        {
+          "use": "usual",
+          "type": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                "code": "DEA",
+                "display": "Drug Enforcement Administration registration number"
+              }
+            ],
+            "text": "DOCDEA"
+          },
+          "system": "urn:ietf:rfc:3986",
+          "value": "234234123",
+          "period": {
+            "start": "2013-11-08T00:00:00.000-06:00"
+          }
+        }
+      ],
+      "active": true,
+      "name": [
+        {
+          "use": "usual",
+          "text": "PCTAUT, OrdersPhy3",
```

Comment:
We may want to use an example with a name that's more representative of what you would see in a production environment.

Reply:
Went with Rory Williams
https://github.com/cerner/fhir.cerner.com/pull/252/commits/114b72f1ff982efcb5315405c67849dae19fcfc8#diff-b7b6f9bc4e73a723f2d414890a03bde8L13